<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilesMenu</class>
 <widget class="QWidget" name="ProfilesMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Profiles</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Device Profiles</string>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P1TitleEntry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Not Configured</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P1ResetButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="P2Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P2TitleEntry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Not Configured</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P2ResetButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="P3Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P3TitleEntry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Not Configured</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P3ResetButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="P4Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P4TitleEntry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Not Configured</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P4ResetButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="P5Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P5TitleEntry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Not Configured</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P5ResetButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveProfilesButton">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>340</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelProfilesButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
